src/base/preferences.cpp

```diff
@@ -448,6 +448,26 @@ void Preferences::setActionOnDblClOnTorrentFn(int act)
     setValue("Preferences/Downloads/DblClOnTorFn", act);
 }
 
+bool Preferences::deleteAddedTorrents() const
```

Comment:
I can not imagine how to avoid duplicating of the settings keys constants. Can you suggest anything?